src/QirRuntime/README.md

```diff
@@ -11,24 +12,17 @@ This folder contains QIR runtime project, which includes implementation of the
 
 ## Build
 
+### Prerequisites
 The QirRuntime project is using CMake (3.17) + Ninja(1.10.0) + Clang++(11.0.0). Other versions of the tools might work
- but haven't been tested. Only x64 architecture is supported.
+ but haven't been tested. Only x64 architecture is supported.  
+For runing the PowerShell scripts below use the powershell core (`pwsh`),
+e.g. [this one](https://github.com/PowerShell/PowerShell/releases/download/v7.1.2/PowerShell-7.1.2-win-x64.msi), not the inbox PowerShell.
```

Comment:
This links directly to a specific version of the Windows 64-bit installer; I'd suggest linking to the PowerShell installation page, both so that we don't need to update the link when new versions are released, and so that users on other operating systems aren't directed to Windows-only links. E.g.: https://docs.microsoft.com/powershell/scripting/install/installing-powershell.